test/data/testSP.py

```diff
@@ -241,23 +240,23 @@ def testmoments(self):
             for m in moment32:
                 self.assertEquals(m, 0.0)
 
-    @skipForParser('ADF', 'Does not support metadata yet')

```

Comment:
Yeah, maybe just rebase this PR so that the metadata tests are activated in one commit and the symmetry stuff is added in another.